BetterChatNames/BetterChatNames.plugin.js

```diff
@@ -15,7 +15,7 @@ const RemoveDashes = true
 
 var titleObserver
 const DashRegex = new RegExp('-|_', 'g')
-const CapitalRegex = new RegExp(/(?<=(^|[^a-zA-ZÀ-ɏḀ-ỿΑ-Ωα-ω'’]))[a-zA-ZÀ-ɏḀ-ỿΑ-Ωα-ω]/g)
+const CapitalRegex = new RegExp(/(?<=(^|[^a-zA-ZÀ-ɏḀ-ỿ\u0370-\u03ff\u1f00-\u1fff'’]))[a-zA-ZÀ-ɏḀ-ỿ\u0370-\u03ff\u1f00-\u1fff]/g)
```

Comment:
Ah thank you, it is quite hard for me to test support for other languages since I only know English. If you could please change this to characters as opposed to Unicode code point form (for the sake of consistency), I will merge the request
```suggestion
const CapitalRegex = new RegExp(/(?<=(^|[^a-zA-ZÀ-ɏḀ-ỿͰ-Ͽἀ-῾'’]))[a-zA-ZÀ-ɏḀ-ỿͰ-Ͽἀ-῾]/g)
```